### DISCUSSION TITLE
Happy birthday !

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#1807] [General]
Hyprland has one year on github now ! Congratulations Vaxerski, awesome work. Cheers !

**Comment by vaxerski** [comment]
Thanks, I wanna hang myself :)

**Reply by Incognitux** [reply]
most sane C++ developer

**Comment by Incognitux** [comment]
Jokes aside, thanks vaxry for this amazing project!
